projects/fourier/lib/fourier/services/bundle/tuist.rb

```diff
@@ -44,11 +44,13 @@ def call
                   output_zip_path,
                   "tuist",
                   "ProjectDescription.swiftmodule/",
+                  "ProjectDescription.swiftinterface",
```

Comment:
with the updates the `.swiftinterface` will be nested within the `.swiftmodule` - i.e. there will no longer be a standalone `ProjectDescription.swiftinterface`